app/assets/javascripts/discourse/app/app.js

```diff
@@ -33,6 +35,9 @@ const Discourse = Application.extend({
 
   // Start up the Discourse application by running all the initializers we've defined.
   start() {
+    setDefaultOwner(this.__container__);
```

Comment:
I'm not sure why this is necessary. Each environment should already be setting the default owner. Did you get a failure without it?

Reply:
It *sounds* to me like the problem is that the theme code is being called before the app has bootstrapped. Instead of adding `setDefaultOwner` in start I would prefer us have the theme code called in the right order. Is that something you can handle?